SubmissionTest.java

```diff
@@ -292,39 +282,267 @@ public void testGetScriptCountValid()
     @Test
     public void testGetScriptCountInvalid()
     {
-        submissions[1].convertToZip();
-        submissions[1].unZip();
-        submissions[1].parseJSONFile();
+        setUpSubmission(submissions[1]);
 
         int unexpected = 4;
         int actual = submissions[1].getScriptCount();
         assertFalse("should be false", unexpected == actual);
     }
 
     /**
+     * Test getScriptCountForStage, valid.
+     */
+    @Test
+    public void testGetScriptCountForStageValid()
+    {
+        setUpSubmission(submissions[1]);
+
+        int expected = 1;
+        int actual = submissions[1].getScriptCountForStage();
+        assertEquals("should be equal", expected, actual);
+    }
+    
+    /**
+     * Test getScriptCountForStage, valid - stage has no scripts.
+     */
+    @Test
+    public void testGetScriptCountForStageValidEmpty()
+    {
+        setUpSubmission(submissions[0]);
+
+        int expected = 0;
+        int actual = submissions[0].getScriptCountForStage();
+        assertEquals("should be equal", expected, actual);
+    }
+    
+    /**
+     * Test getScriptCountForStage, invalid.
+     */
+    @Test
+    public void testGetScriptCountForStageInvalid()
+    {
+        setUpSubmission(submissions[1]);
+
+        int unexpected = 2;
+        int actual = submissions[1].getScriptCountForStage();
+        assertFalse("should be false", unexpected == actual);
+    }
```

Comment:
I'm pretty sure this unit test is not necessary.  It seems trivial to test that a function's output is not equal to an arbitrary number.